SU-HSDO/suhumsci
# Arch 191 course content

# READY FOR REVIEW

# Summary
- Course content type and importer

# Needed By (Date)
- EOD wednesday if possible (end of sprint)

# Urgency
- low

# Steps to Test

1. checkout branch
2. `composer update --prefer-source`
3. `blt drupal:sync --site=archaeology`
4. go to `/admin/config/system/keys` and click “add Key”
5. in the “key value” field put “http://yourdomain.local/api/hs_courses”
6. go to `admin/config/development/configuration/key-overrides/add` and set up the configuration override like the screenshot here: https://d.pr/free/i/V9z8rh
7. `drush @archaeology.local cron`
8. validate course content got created. theres should be about 124 courses.
9. validate course data matches [explore courses](http://explorecourses.stanford.edu/search?q=ARCHLGY&view=xml-20140630&collapse=&filter-departmentcode-ARCHLGY=on&filter-coursestatus-Active=on&filter-catalognumber-ARCHLGY=on)
10. attempt to view a course when logged out. it should bounce you to the explore courses page.
